### ISSUE TITLE
Player disconnect messages don't display client number

### ISSUE THREAD

**Issue by Razish** [Razish/japp#332]
for duplicate names (i.e. multiple Padawan players) it should show the name we substitute it with

**Comment by ensiform** [comment]
If the message is coming from the server engine, it will not show because its not updated in the engine I think.

**Comment by dpadgett** [comment]
the name substitution doesn't work anyway.  the numbers all get cleared on
map change and then everyone has the same name again.

**Comment by ensiform** [comment]
Right.  But the number gets added in ClientUserinfoChanged but never sent back to the engine which is why the disconnect message only says the original name.

**Comment by ensiform** [comment]
I think mods should be able to control the message to be honest.

Maybe a flag in new api that says if mod handles it or not.
